pixart.js

```diff
@@ -1 +1,31 @@
+$("#form").submit(setColor);
 
+
+// Sets the color that user will paint with
+// ****************************************
+function setColor() {
+  event.preventDefault();
+  var $userInput = $("#color-field").val();
+  var $colorBox = $(".brush").eq(0);
+  $($colorBox).css("background-color", $userInput);
+}
+
+
+// Takes color from user and makes the divs bground that color
+// ***********************************************************
+function squareColor() {
```

Comment:
This name is a bit ambiguous to me... I prefer function names to be action verbs where possible... e.g. `paintSquare()`.